lnbits/bolt11.py

```diff
@@ -23,6 +23,7 @@ class Route(NamedTuple):
 
 class Invoice(object):
     payment_hash: str
+    checking_id: Optional[str] = None
```

Comment:
`checking_id` should not be in invoice object. Why is this here?

Reply:
your are using it on fakewallet            
```python
 hasattr(invoice, "checking_id")
 invoice.checking_id is not None
```

Reply:
Oops that's very bad practice. Please do not change the model.